.github/workflows/ci.yml

```diff
@@ -209,11 +232,14 @@ jobs:
       - name: Download test report
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093 # v4.3.0
         with:
-          pattern: test-report
+          pattern: test-report-linux
+
+      - name: List downloaded files
```

Comment:
Is this supposed to stay for diagnostics?

Reply:
Yes, this is the residual and needs to be removed.

Reply:
[Addressed review comments](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/pull/408/commits/5f5e2b639c25d227b81ee445edb07880f2f2521b)